### ISSUE TITLE
Feature: Native WKT reader/writer

### ISSUE THREAD

**Issue by Maxxen** [duckdb/duckdb-spatial#11]
We currently rely on GEOS for writing and reading WKT, which requires an unnecessary copy and allocation step to convert back and forth between DuckDB and GEOS geometries. 

We should implement our own WKT parser/writer that can use arena allocation.

Since we also only support of a subset of WKT (no extended WKT) this would also give us more control and ensure we don't accidentally import geometries we can't support - basically throw at the parsing step.

**Comment by Maxxen** [comment]
Yes! "WKT" and "Text" is the same thing. Use ST_GeomFromText(VARCHAR). You can pass an 'ignore_invalid := <bool> flag too to get nulls instead of error should the geom be invalid.

**Comment by marklit** [comment]
Okay, thanks.
